Sources/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -2,15 +2,7 @@
 #import <CoreMedia/CoreMedia.h>
 #import <UIKit/UIKit.h>
 
-#if MUX_COCOAPODS
-#if __has_include("MUXSDKStats-Swift.h")
-#import "MUXSDKStats-Swift.h"
-#else
-#import <MUXSDKStats/MUXSDKStats-Swift.h>
-#endif
-#else
 @import MUXSDKStatsInternal;
-#endif
```

Comment:
CocoaPods is now built as another project that consumes our SPM library, so no more trying to keep both systems happy.